ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -1,12 +1,64 @@
 # global
 import numpy as np
 from hypothesis import given, strategies as st
+import pytest
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    statistical_dtype_values,
+)
 from ivy_tests.test_ivy.helpers import handle_cmd_line_args
 
 
+def is_broadcastable(shp1, shp2):
+    for a, b in zip(shp1[::-1], shp2[::-1]):
+        if a == 1 or b == 1 or a == b:
+            pass
+        else:
+            return False
+    return True
+
+
+@pytest.mark.filterwarnings("ignore: dist")
+@handle_cmd_line_args
+@given(
+    dtype_and_x=statistical_dtype_values(function="dist"),
```

Comment:
Please use the helper function dtype_and_values() to generate the data :)

In this case you should use the following settings
```
helpers.dtype_and_values(
available_dtypes = helpers.dtypes("numeric")
num_arrays = 2,
shared_dtype=True,
allow_inf=False,
)
```
This should generate 2 arrays of similar shape that you can use instead of x and y. :)

This code ^^ should generally be correct but please feel free to modify it to suit the function.

Reply:
Yeah, It worked well, I only had to change `helpers.dtypes`  to `helpers.get_dtypes` as I got an error saying `module 'ivy_tests.test_ivy.helpers' has no attribute 'dtypes'`.